test/util.py

```diff
@@ -161,15 +163,19 @@ def assert_contract_code(address):
     # just checking key equality
     assert_equal(sorted(code.keys()), ["abi", "bytecode"])
 
-def assert_contract_definition(address):
+def assert_contract_definition(address, contract_path):
     """Asserts the content of the contract definition of a contract at address."""
     output = my_run([
         "starknet", "get_full_contract",
         "--contract_address", address
     ])
-    contract_definition = json.loads(output.stdout)
-    # just checking key equality
-    assert_equal(sorted(contract_definition.keys()), ["abi", "entry_points_by_type", "program"])
+    contract_definition: ContractDefinition = ContractDefinition.load(json.loads(output.stdout))
+
+    with open(contract_path, encoding="utf-8") as contract_file:
+        loaded_contract = json.load(contract_file)
+        loaded_contract_definition: ContractDefinition = ContractDefinition.Schema().load(loaded_contract)
```

Comment:
If `ContractDefinition` approach is kept, why not use the same approach as above (without explicitly invoking `Schema`)?

Reply:
All changes were addressed in [23c20b9](https://github.com/Shard-Labs/starknet-devnet/pull/39/commits/23c20b946d8f0f13cb241b90cbb3ad0d158e27c9)